src/remote-config/condition-evaluator-internal.ts

```diff
@@ -173,18 +173,8 @@ export class ConditionEvaluator {
   }
 
   static hashSeededRandomizationId(seededRandomizationId: string): bigint {
-    // For consistency with the Remote Config fetch endpoint's percent condition behavior
-    // we use Farmhash's fingerprint64 algorithm and interpret the resulting unsigned value
-    // as a signed value.
-    let hash64 = BigInt.asIntN(64, farmhash.fingerprint64(seededRandomizationId));
-
-    // Manually negate the hash if its value is less than 0, since Math.abs doesn't
-    // support BigInt.
-    if (hash64 < 0) {
```

Comment:
Thinking (no action req'd): we're now directly instantiating the bigint with the hash, rather than coercing it to behave like a 64bit signed Java long, so it will always be positive.